project-bwk.yml

```diff
@@ -0,0 +1,184 @@
+name: BitwardenKit
+fileGroups:
+  - Configs
+configs:
+  Debug: debug
+  Release: release
+options:
+  createIntermediateGroups: true
+  deploymentTarget:
+    iOS: 15.0
+    watchOS: 8.0
+  groupSortPosition: top
+  usesTabs: false
+  indentWidth: 4
+  tabWidth: 4
+settings:
+  MARKETING_VERSION: 2024.6.0    # Bump this for a new version update.
+  CURRENT_PROJECT_VERSION: 1
+include:
+  - path: Configs/local-sdk.yml
+    relativePaths: false
+    enable: ${LOCAL_SDK}
+packages:
+  BitwardenSdk:
+    url: https://github.com/bitwarden/sdk-swift
+    revision: 94fd09f0b73367071bc033987a1962fce85f5c76
+    branch: unstable
```

Comment:
🤔 Do we need the `BitwardenSdk` for the kit? I think the ones listed here doesn't currently use it.